.github/workflows/ci.yml

```diff
@@ -222,6 +223,17 @@ jobs:
       project-name: Component[OpenTelemetry.Instrumentation.Cassandra]
       code-cov-name: Instrumentation.Cassandra
 
+  build-test-instrumentation-confluentkafka:
+    needs: detect-changes
+    if: |
+      contains(needs.detect-changes.outputs.changes, 'instrumentation-confluentkafka')
+      || contains(needs.detect-changes.outputs.changes, 'build')
+      || contains(needs.detect-changes.outputs.changes, 'shared')
+    uses: ./.github/workflows/Component.BuildTest.yml
+    with:
+      project-name: Component[OpenTelemetry.Instrumentation.ConfluentKafka]
+      code-cov-name: Instrumentation.ConfluentKafka          
```

Comment:
Please define `unittests-Instrumentation.ConfluentKafka` in here:

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/cd7e8d3cb6cb73c9dcb1398ac5914081d7578524/.github/codecov.yml#L70

Sorry there is a lot of ceremony scattered around 😄

Reply:
done